frontend/src/store/index.js

```diff
@@ -465,7 +464,7 @@ const getters = {
             expirationDate,
             vendorName,
             icon: vendorName,
-            needsLicense
+            vendorHint: findVendorHint(state.cfg.vendorHints, vendorName)
```

Comment:
The `vendorHint` only depends on the vendor name and should not be determined outside of the iteratee.
```js
        const vendorName = vendorNameFromImageName(machineImage.name)
        const vendorHint = findVendorHint(state.cfg.vendorHints, vendorName)

        return map(versions, ({ version, expirationDate, cri, classification }) => {
          return decorateClassificationObject({
            key: name + '/' + version,
            name,
            version,
            cri,
            classification,
            expirationDate,
            vendorName,
            icon: vendorName,
            vendorHint
          })
        })
```

Reply:
done